test/unit/high-security.test.js

```diff
@@ -243,9 +243,11 @@ describe('high security mode', function() {
     })
 
     it('should not affect addCustomAttribute if high_security is off', function() {
-      agent.config.high_security = false
-      var success = api.addCustomAttribute('key', 'value')
-      should.not.exist(success)
+      helper.runInTransaction(agent, () => {
```

Comment:
I changed other failure cases to return `false` which then failed this test which was relying on those silently failing. Running in a transaction makes everything legit for the test.